more/free-programming-cheatsheets.md

```diff
@@ -92,6 +93,11 @@
 * [What is Artificial Intelligence?](https://intelligencereborn.com/ArtificialIntelligence.html) IntelligenceReborn (HTML)
 
 
+### Bench
+
+* [Bench CLI Cheatsheet](https://frappeframework.com/files/Bench%20Cheatsheet%20v1.1.pdf) (PDF)
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.